content/applications/hr/frontdesk.rst

```diff
@@ -0,0 +1,318 @@
+=========
+Frontdesk
+=========
+
+The Odoo *Frontdesk* application is a way for visitors to check in to a building or location and
+alert the person they are seeing that they have arrived. Additionally, they can request a
+pre-configured beverage, to be brought to them while they wait. This application is ideal for
+businesses that do not have someone working at a reception desk, or perhaps do not have a lobby or
+waiting area available to guests and visitors.
+
+Configuration
+=============
+
+The first item to be configured to use the *Frontdesk* application is the station or stations,
+followed by the drinks that are offered.
+
+Stations
+--------
+
+A **Station** can be thought of as any location where someone can sign-in and wait for an employee.
+This is typically some form of waiting room, such as a lobby. Each station has a kiosk where
+visitors check in. There is a minimum of one station that must be configured to use the *Frontdesk*
+application, but there is no limit to how many stations that can created and configured.
+
+To create a station, navigate to :menuselection:`Frontdesk application --> Configuration -->
+Stations` and click :guilabel:`New`. A blank frontdesk form loads. Enter the following information
+on the form:
+
+- :guilabel:`Frontdesk Name`: enter the name for the specific Frontdesk location. This should be
```

Comment:
```suggestion
- :guilabel:`Frontdesk Name`: enter the name for the specific frontdesk location. This should be
```
It should be lowercase, right? Just basing this on what I'm seeing below, feel free to ignore if I'm wrong though